website/content/api-docs/jobs.mdx

```diff
@@ -595,6 +595,7 @@ $ curl \
             "SizeMB": 300,
             "Sticky": false
           },
+          "MaxClientDisconnect": null,
```

Comment:
Rather than having the value be null here, it might be nice if our API docs could show a real example value at least oonce.

Reply:
Updated in next push.